src/oracles/ChainlinkOracle.sol

```diff
@@ -54,13 +75,21 @@ contract ChainlinkOracle is IOracle, Ownable {
     /// @inheritdoc IOracle
     function price(address token) external view returns (bool success, uint256 priceX96) {
         IAggregatorV3 chainlinkOracle = oraclesIndex[token];
+        success = true;
+        uint256 oraclePrice;
+        uint256 updatedAt;
         if (address(chainlinkOracle) == address(0)) {
-            return (false, 0);
+            success = false;
```

Comment:
If chainlink is not configured, current contract expects oracle owner to submit price updates every ~27 seconds.

I don't think we will ever use oracle in such configuration, so we can keep the `return (false, 0)` here.